programs/tests/c-tests/add/add.c

```diff
@@ -15,6 +15,7 @@ int main() {
     uint32_t z = add(x, y);
     _fp_print(z);
 
+    assert(z == 3);
```

Comment:
nit: any reason not to add asserts to the other two adds as well?

Reply:
There is only a single add statement here?

Reply:
oh sorry I misread; I thought the prints corresponding to x=1 and y=2 were also adds